reattacher/reattacher_main.c

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (C) 2020 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#include <getopt.h>
+#include "accelerator/config.h"
+#include "cclient/api/core/core_api.h"
+#include "cclient/api/extended/extended_api.h"
+#include "common/model/bundle.h"
+#include "storage/ta_storage.h"
+#include "time.h"
```

Comment:
Avoid using `#include "time.h"`. Instead, use `#include "<somewhere>/time.h"`, otherwise both compiler and static analyzer might confuse.